### ISSUE TITLE
Fully upport request and response headers

### ISSUE THREAD

**Issue by axnsan12** [axnsan12/drf-yasg#123]
* [ ] it should be easier to manually specify headers on responses
* [ ] paginator and filter inspectors should be able to add request and response headers
* [ ] ``AcceptHeaderVersioning`` should be detected and handled

**Comment by jhtimmins** [comment]
Is this still relevant? If so I'm interested in working on it.

**Comment by axnsan12** [comment]
Yeah, none of the points in this issue have improved significantly.
